PostHog/PostHogContext.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+import LocalizedDeviceModel
```

Comment:
Can we vendor this extension instead of depending on it directly? this avoids conflicts with dependencies.
We can vendor the class here https://github.com/PostHog/posthog-ios/tree/main/PostHog/Utils and add the license attribution as a comment.

Reply:
I would not recommend to vendor the extension because it depends of a localised file to get the "commercial" model name from the device identifier and because each time a new model is released by Apple, you will miss it.

Reply:
BTW if you want to embed it, that ok for the attribution.

Reply:
That's why we have https://github.com/PostHog/posthog/issues/18025
The goal is to not depend on SDK updates
If this is important for you right now, you can use the props sanitizer https://github.com/PostHog/posthog-ios/blob/b0c6c1847228ce1fb78438be21cfe39cddc55254/PostHog/PostHogConfig.swift#L32C22-L32C41
read the current value, check what's the model name, and update with the human readable model name.